events/subscriptions/subscription_change.md

```diff
@@ -0,0 +1,36 @@
+# Subscribe
+
+Fire whenever a user changes their subscription (other than a subscribe or unsubscribe event).
+
+## HTML Data Attributes
+
+TBD
```

Comment:
nit, but suggest leaving this out until we know which (if any) attributes will be required